design-documents/graph-ql/coverage/CartAddressOperations.graphqls

```diff
@@ -96,15 +105,24 @@ type CartAddressRegion {
     label: String
 }
 
-enum AdressTypeEnum {
-    SHIPPING
-    BILLING
+type SelectedShippingMethod {
+    carrier_code: String
+    method_code: String
+    carrier_title: String
+    method_title: String
+    amount: Money
+    base_amount: Money
 }
 
-type CheckoutShippingMethod {
-    code: String
-    label: String
-    free_shipping: Boolean!
+type AvailableShippingMethod {
+    carrier_code: String!
+    carrier_title: String!
+    method_code: String
+    method_title: String
     error_message: String
-    # TODO: Add more complex structure for shipping rates
+    amount: Money!
+    base_amount: Money
```

Comment:
Why `base_amount` is exposed via GraphQL?

Reply:
Synchronised with latest version of the schema.
The `base_amoun` is deprecated field and I have removed it from the schema.